libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2093,6 +2093,54 @@ void GCS_MAVLINK::send_hwstatus()
         0);
 }
 
+bool GCS_MAVLINK::try_send_gps_message(const enum ap_message id)
+{
+    AP_GPS *gps = get_gps();
+    if (gps == nullptr) {
+        return true;
+    }
+
+    bool ret = true;
+    switch(id) {
+    case MSG_SYSTEM_TIME:
+        CHECK_PAYLOAD_SIZE(SYSTEM_TIME);
+        send_system_time(*gps);
+        ret = true;
+        break;
+    case MSG_GPS_RAW:
+        CHECK_PAYLOAD_SIZE(GPS_RAW_INT);
+        send_gps_raw(*gps);
```

Comment:
Humm, I said to remove it in the function for a reason: every other message will do it here and if there is no space to send it, the *try_send_message* method will return false. Removing it here and let it be in the *send_gps_raw* won't do the same - unless you keep the boolean return and use it, which I think is something you are always trying to avoid.